build/make-initrd.sh

```diff
@@ -9,6 +9,18 @@ cp iptables /tmp/hyperstart-rootfs
 cp libm.so.6 /tmp/hyperstart-rootfs/lib64/
 tar -xf modules.tar -C /tmp/hyperstart-rootfs/lib/modules
 
+# create symlinks to busybox and iptables
+for bin in ${BUSYBOX_BINARIES}
+do
+	mkdir -p /tmp/hyperstart-rootfs/`dirname ${bin}`
+	ln -s /busybox /tmp/hyperstart-rootfs/${bin}
+done
```

Comment:
You are trying to move the symblink from *.c to the Makefile&*.sh which is much more appropriate for building a distro(hyperstart/hyper-initrd can be considered as a small distro)

But I don't think you will use the same build tools(Makefile&*.sh) to build a different distro(a custom image), so it seems we don't need hostpath in these Makefile&*.sh.

Correct me if I was wrong and tell me the way you re-use the build tools.